api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -157,17 +154,17 @@ public static String getOpenmrsProperty(String property) {
 	 * @see #APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY
 	 * @see OpenmrsUtil#getApplicationDataDirectory()
 	 */
-	public static String APPLICATION_DATA_DIRECTORY_FALLBACK_UNIX = "/var/lib";
+	public static final String APPLICATION_DATA_DIRECTORY_FALLBACK_UNIX = "/var/lib";
 	
-	public static String APPLICATION_DATA_DIRECTORY_FALLBACK_WIN = System.getenv("appdata");
+	public static final String APPLICATION_DATA_DIRECTORY_FALLBACK_WIN = System.getenv("appdata");
 	
 	/**
 	 * The name of the runtime property that a user can set that will specify where openmrs's
 	 * application directory is
 	 * 
 	 * @see #APPLICATION_DATA_DIRECTORY
 	 */
-	public static String APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY = "application_data_directory";
+	public static final String APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY = "application_data_directory";
 	
```

Comment:
`AUTO_UPDATE_DATABASE_RUNTIME_PROPERTY` should also be final

Reply:
@teleivo Thanks. I will make the necessary changes.